spk/pyload/src/dsm-control.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+# Package
+PACKAGE="pyload"
+DNAME="pyLoad"
+
+PIDFILE="/var/run/pyload.pid"
+PYLOADCORE="/usr/local/python/bin/python /usr/local/pyload/share/pyload/pyLoadCore.py --pidfile=${PIDFILE}"
+export PATH="/usr/local/pyload/bin:$PATH"
+
+case $1 in
```

Comment:
I'd prefer the same structure as we use in other packages: split off the various sections into their own functions, and call those functions.